war/src/main/less/abstracts/theme.less

```diff
@@ -151,23 +160,38 @@
   --link-dark-font-weight: 600;
 
   // Pane
-  --pane-header-color: #eee;
-  --pane-header-border-color: #f3f3f3;
-  --pane-border-color: var(--very-light-grey);
+  --pane-border-width: 1px;
+  --pane-header-text-color: var(--white);
+  --pane-header-bg: var(--dark-grey);
+  --pane-header-border-color: var(--light-grey);
+  --pane-header-font-weight: bold;
+  --pane-border-color: var(--light-grey);
   --pane-text-color: #3b3b3b;
   --pane-link-color: black;
   --pane-link-color--visited: black;
 
-  // TabBar
-  --tab-bar-border-color: var(--very-light-grey);
-  --tab-bar-text-color: #999;
-  --tab-bar-bg--checked: #eee;
-  --tab-bar-border-color--checked: var(--very-light-grey);
-  --tab-bar-text-color--checked: #000;
-  --tab-bar-bg-color--hover: #eee;
-  --tab-bar-add-text-color: #0460d1;
-  --tab-bar-add-bg-color--hover: #d8efff;
-  --tab-bar-add-border-color: #d8efff;
+  // TabBar links
+  --tab-link-border-radius: 4px;
+  --tab-link-border-width: 2px;
+  --tab-link-padding-x: 0.75rem;
```

Comment:
do the paddings need to be exposed in the API?

Reply:
I put them there because it made it easier for me to iterate the styles. Do you think they should be pruned?

Reply:
I think we need to decide how far we want to go with the CSS vars